### DISCUSSION TITLE
Prevent duplicates

### DISCUSSION THREAD

**Discussion by kmander** [Kovah/LinkAce#565] [Ideas & Feature Requests]
It'd be great to prevent the entry/import of any duplicate URLs.

**Comment by Kovah** [comment]
Could you please elaborate more on this issue? Currently it's not possible to enter a link with the same URL twice.

Regarding stuff like `https://example.com/my-blog-post.html` and `https://example.com/my-blog-post.html#`: logic to handle this will never happen.
URLs and their usage in website are way too complex to properly detect if the second link with just a `#` more _really_ is a duplicate, or loads another page, or does something specific on the page which makes it somewhat unique.
